### ISSUE TITLE
"Stuff MoFo Customers Say:" sharing real feedback / insights from the communities we serve

### ISSUE THREAD

**Issue by valianttry** [MozillaFoundation/MoFoLondon#21]
## Description:

CRAZY IDEA: Share quotes, surveys, real-talk from our constituents. We do a lot of research, surveys, and feedback loops these days... what are we learning?

**Comment by ldecoursy** [comment]
cc @OpenMatt @adamlofting Interested in fleshing this out as it relates to surveys underway, stories as data, etc?

**Comment by OpenMatt** [comment]
yeah we have a bunch of this from the network survey for MLN.
and from desk.com. I _love_ the idea of thinking about ways we can pull patterns out.

would be great to share some of the stories we got through desk.com on the Advocacy side as well.

cc @samanthaburton as she is interested in this kind of content for State of the Web as well.

**Comment by valianttry** [comment]
@OpenMatt and @samanthaburton and @stephaniemcv and @mmmari I'm going to schedule a quick chat for us to talk about some of the findings we might be able to share here.

**Comment by ldecoursy** [comment]
Is this connected to #13? 

If not, we'll need to add to the schedule somewhere.

**Comment by samanthaburton** [comment]
It isn't right now - #13 is focused on sharing & aligning story gathering processes, while it looks like this one is sharing & analyzing data we already have.

**Comment by ldecoursy** [comment]
I might suggest we do this during Wed or Thurs elective time. Any preferences?

**Comment by valianttry** [comment]
Here is a quick hits list of stuff we can share (this is no doubt forgetting good stuff):
- Desk.com feedback data: https://docs.google.com/document/d/1zpCE_Csbttnb5reIWBJVEaqO4QW6SG_qkcKXPUN-D2M/edit
- Why do donors donate to Mozilla: https://fundraising.mozilla.org/why-did-you-decide-to-donate-today/
- Make up of our donors (demographics) charts at: https://fundraising.mozilla.org/
- Demographics of our email list: https://docs.google.com/presentation/d/1rM2d2VZwoheR4wgWLsZs30OUpH8BF9A-lXjia1mllMo/edit#slide=id.gba306b5ca_0_46
  ![survey_results_-_google_slides](https://cloud.githubusercontent.com/assets/9830371/15843224/2aa0ad68-2c15-11e6-8c51-68e237e80063.png)

@stephaniemcv has some data too!

**Comment by xmatthewx** [comment]
This is not currently listed in the schedule spreadsheet or app. Is that okay? Should we close this?

**Comment by valianttry** [comment]
Yes -- in the meeting you couldn't make we decided this should wait until Hawaii. No one had enough capacity to pull the data together ahead of time.

Thanks @OpenMatt
